.eslintrc

```diff
@@ -4,20 +4,30 @@
     "strict": 0,
     "no-underscore-dangle": 0,
     "no-unused-vars": 0,
-    "curly": 0,
     "no-multi-spaces": 0,
     "key-spacing": 0,
     "no-return-assign": 0,
```

Comment:
Let's have this on as well. Don't really see any point of assignments in return statements.